src/go/rpk/pkg/cli/cmd/redpanda/stop.go

```diff
@@ -63,14 +64,14 @@ running.`,
 	return command
 }
 
-func executeStop(
-	fs afero.Fs, mgr config.Manager, configFile string, timeout time.Duration,
-) error {
-	conf, err := mgr.ReadOrFind(configFile)
+func executeStop(fs afero.Fs, cmd *cobra.Command, timeout time.Duration) error {
+	p := config.ParamsFromCommand(cmd)
```

Comment:
I think `ParamsFromCommand` should go into the `Run` function, and what's below should stay here.